### ISSUE TITLE
Error -20 iOS13.4 iPhone8

### ISSUE THREAD

**Issue by Ssiswent** [checkra1n/BugTracker#1194]
**Tell us about your setup:**
1. What iDevice are you using?   **iPhone 8**
2. On what version of iOS is it?  **iOS13.4 or iOS13.4.1**
3. What version of checkra1n are you using?   **0.10.1**
4. What is your host system (OS version? Hackintosh? VM? etc.)?   **Mac mini (Late 2014) 10.15.3**
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?    **USB-A**

**What are the steps to reproduce the issue?**
1. After I update to iOS13.4 or 13.4.1
2. 
3.
...

**What do you expect, and what is happening instead?**

Error -20.

**Does the issue also occur if you enable Safe Mode?**

Yes.

**Any other info, error logs, screenshots, ...?**

![1](https://user-images.githubusercontent.com/37449310/79040908-790d4f80-7c1e-11ea-9c65-89db2a0ce4bf.jpg)
![2](https://user-images.githubusercontent.com/37449310/79040913-7d396d00-7c1e-11ea-9dc1-744ad398f4ee.jpg)
![3](https://user-images.githubusercontent.com/37449310/79040923-89bdc580-7c1e-11ea-89ca-454253421b51.jpg)

**Comment by ThinkHap** [comment]
same at iPhone X   13.4.1

**Comment by Siguza** [comment]
The output on your screen (thanks for that btw) seems to suggest it's attempting an upgrade... is it possible you interrupted an OTA update there, by any chance?  
Can you boot normally into unjailbroken mode?  
And when the device is in the state pictures above, could you try if you can still SSH in?

**Comment by Ssiswent** [comment]
> The output on your screen (thanks for that btw) seems to suggest it's attempting an upgrade... is it possible you interrupted an OTA update there, by any chance?
> Can you boot normally into unjailbroken mode?
> And when the device is in the state pictures above, could you try if you can still SSH in?

Yes, when I was on iOS13.3, I used iCleaner to block OTA update. And I tried to update to 13.4.1, then I found that I must open OTA update in iCleaner. But after that, I failed to download 13.4.1 for serveral times. However, I successfully updated to 13.4.1 at last.(System is totally fine all the time.)
I can boot into unjailbroken mode, that works fine. I tried iOS13.4, iOS13.4.1, checkra1n 0.9.9, 0.10.0, 0.10.1 but they all failed. They all stuck in the same place, "faied to find boot-hash-manifest snapshot".
And I don't really understand your last question, because I don't know how to check if I can SSH in.
Last but not least, thanks for your answering.

**Comment by codeVerine** [comment]
I'm also facing the exact same issue on my iPhone X 13.5. It would be much appreciated if the team can have a look at this issue. I love how stable checkra1n JB is and reluctant to switch to other JBs. This issue is keeping me away from enjoying current JB scene. Thanks.
